src/AltinnCore/Designer/Controllers/RuntimeAPIController.cs

```diff
@@ -340,7 +340,7 @@ public ServiceState UpdateCurrentState(string org, string service, string develo
         {
             string workflowFullFilePath = _settings.GetWorkflowPath(org, service, developer) + _settings.WorkflowFileName;
             string workflowData = System.IO.File.ReadAllText(workflowFullFilePath, Encoding.UTF8);
-            string serviceStatePath = $"{_settings.GetTestdataForPartyPath(org, service, developer)}{partyId}/{instanceId}.state.json";
+            string serviceStatePath = $"{_settings.GetTestdataForPartyPath(org, service, developer)}{partyId}/{instanceId}/{instanceId}.state.json";
```

Comment:
why are we changing the path? And is this really the only place its necessary to change this?

Reply:
So now we have this structure, Testdataforparty/party/instanceid/instance meta data doc, state doc, 
then Testdataforparty/party/instanceid/instance/data where all the formdataxml, attachments will be stored. I had already updated the path in other places as part of #1374 .